internal/unionstore/memdb.go

```diff
@@ -837,12 +836,8 @@ func (n *memdbNode) setBlack() {
 }
 
 func (n *memdbNode) getKey() []byte {
```

Comment:
Are there any tests against this function?

Reply:
Current UT cases in [`memdb_test.go`](https://github.com/tikv/client-go/blob/1de9d430a21130f4854e4164c3459066d37977cf/internal/unionstore/memdb_test.go#L65) has covered this method.

E.g. `TestGetSet` -> `p.Get(..)` -> `db.traverse(..)` -> `getKey`.

And the coverage result also prove that (`getKey` is green):

![img_v3_02df_b2c1ed7e-5567-4aa8-b3ce-9e8cba385bag](https://github.com/user-attachments/assets/06abf6e6-ab28-4736-8199-dbbbc6eb1342)